### PR TITLE
Update ID structure for sea transport

### PR DESCRIPTION
**Pull request by greenercode**
Review all sea transport emission factors

**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Sea transport 

**What?**

For all EFs incurrently in the sea transport category:

IDs updated to adhere to the following machine-readable structure:
sea_transport-ship_type [e.g. "bulk_carrier"]- route_type_[e.g.intra_mediterranean]-ship_load_[e.g. "above-8000-teu"]-ship_length_[e.g. 163m]-fuel_source_[e.g. "unknown" or "diesel"]-activity_[e.g. "fuel_combustion", "upstream" or "fuel_combustion_and_upstream" or "unknown"]

Name field to be concise, easily read plain English which is as consistent as possible across EFs regardless of source
Description field to provide further detail regarding the specifics of the EF, including life-cycle activity details and specifics of applicability
Updates description field

**Why?**

This is part of a larger refactoring of IDs across the OEFDB to ensure EFs adhere to a clear standard structure per category and across sources, allowing them to be better understood and compared both by humans and machines.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -295,7 +295,7 @@ Transport,Road Travel,bus-mid-agglomeration,Bus (Medium Agglomeration),kg-CO2e/p
...
+Transport,Sea Transport,sea_freight-vessel_type_cargo-route_type_unknown-vessel_length_unknown-tonnage_below_or_equal_to_10000t-fuel_source_heavy_fuel_oil_or_marine_gas_oil-activity_upstream_and_fuel_combustion,Cargo ship using heavy fuel or marine gas oil ≤ 10000t,kg-CO2e/tkm,0.0191,60,ADEME,2020,FR,2021/08/20,"Emission intensity for cargo ship of up to 10000t using heavy fuel oil or marine gas oil. Cargo ships are commercial vessels, intended for the transport of general cargo. The data are pro-posed by tonnage capacity. Chemical tankers belong to this category. As the data is relatively close between HFO and MGO, it was decided to include in the Base Carbone® only an average data, combining these two fuels, in proportion to their relative use. Provided by the GT Transport Base Carbone & GLEC Framework. Published by the Agence de l'Environnement et de la Maîtrise de l'Énergie in May 2020.",https://www.bilans-ges.ademe.fr/en/accueil
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:298]: Does the source define what type of upstream emission it is? Sometimes ADEME defines that.

**Review by maryarbab — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -281,26 +281,27 @@ Energy,Fuel,bio-petrol,Biopetrol,kg-CO2e/l,0.01402,,BEIS,2021,GB,2021/07/23,Emis
...
-Transport,Sea Transport,bulk-carrier-between-100000-and-199999dwt,Bulk carrier between 100.000 and 199.999 dwt,kg-CO2e/tkm,0.003042,,MfE,2020,NZ,2021/09/10,"Emission intensity of a bulk carrier. Between 100000 and 199999 dwt. The dwt value corresponds to deadweight tonnage, a measure of how much weight a ship can carry. International shipping. New Zealand. Published by the Ministry for the Environment of New Zealand in 2020.",https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/
+Transport,Sea Transport,sea_freight-vessel_type_bulk_carrier-route_type_unknown-vessel_length_unknown-tonnage_above_200000dwt-fuel_source_unknown-activity_fuel_combustion,Bulk carrier > 200000 dwt,kg-CO2e/tkm,0.002535,,MfE,2020,NZ,2021/09/10,"Emission intensity for bulk carrier. Above 200000 dwt. The dwt value corresponds to deadweight tonnage, a measure of how much weight a ship can carry. International shipping. New Zealand. Published by the Ministry for the Environment of New Zealand in 2020.",https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:284]: Can you please remove " in the description?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -281,26 +281,27 @@ Energy,Fuel,bio-petrol,Biopetrol,kg-CO2e/l,0.01402,,BEIS,2021,GB,2021/07/23,Emis
...
+Transport,Sea Transport,sea_freight-vessel_type_bulk_carrier-route_type_unknown-vessel_length_unknown-tonnage_average-fuel_source_unknown-activity_fuel_combustion,Bulk carrier (average),kg-CO2e/tkm,0.00578,,MfE,2020,NZ,2021/09/10,Emission intensity for bulk carrier (average). International shipping. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:285]: What does "average" refer to here?

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (greenercode) [reply OpenEmissionFactorsDB.csv:298]: Unfortunately not. There is only upstream (fuel) and combustion (fuel).

**Review by patstirling — CHANGES_REQUESTED** [review]
Still quite a lot of "s

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1196,8 +1197,8 @@ Transport,Vehicle,td-vans-average-battery-ev,T&D Vans Average Battery EV,kg-CO2e
...
+Transport,Sea Transport,sea_freight-vessel_type_tanker-route_type_amsterdam_rotterdam_antwerp-vessel_length_135m-tonnage_unknown-fuel_source_unknown-activity_fuel_combustion,Tanker vessel 135m,g-CO2e/tkm,22,,STC-Nestra B.V.,2019,EU,2021/09/21,Emission intensity for tanker vessel (135 meters). Amsterdam-Rotterdam-Antwerp route. Published by STC-NESTRA B.V in 2018.,https://www.smartfreightcentre.org/pdf/GLEC-report-on-GHG-Emission-Factors-for-Inland-Waterways-Transport-SFC2018.pdf
+Transport,Sea Transport,sea_freight-vessel_type_transhipment-route_type_unknown-vessel_length_unknown-tonnage_ambient-vessel_length_unknown-fuel_source_unknown-activity_fuel_combustion,Transshipment (ambient),kg-CO2e/t,1.2,,FRAUNHOFER,2018,EU,2021/09/22,Emission intensity for transshipment (ambient). Amsterdam-Rotterdam-Antwerp route. Published by the Fraunhofer Institute in 2019.,https://www.iml.fraunhofer.de/content/dam/iml/en/documents/OE%20310/Emission_%20intensity_factors_logistics_buildings_FraunhoferIML_052019.pdf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1201]: duplicate "vessel_length_unknown" in the ID

**Review by patstirling — APPROVED** [review]
SHIP IT!

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1231,17 +1232,16 @@ Transport,Vehicle,vans-class-3-battery-ev,Vans Class 3 Battery EV,kg-CO2e/km,0.0
...
+Transport,Sea Transport,sea_freight-vessel_type_inland_goods-route_type_unknown-vessel_length_unknown-tonnage_500t-fuel_source_diesel-activity_fuel_combustion,Inland freight ship goods,kg-CO2e/tkm,0.0392,,UBA,2020,DE,2021/09/15,Emission intensity for inland waterway vessel with diesel engine. Provided by the Öko-Institut.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={D779AB69-5CAE-453C-B7E8-4CBA9060B24F}
+Transport,Sea Transport,sea_freight-vessel_type_panamax-route_type_unknown-vessel_length_unknown-tonnage_50000t-fuel_source_bunker_c_oil-activity_fuel_combustion,Cargo freighter (Panamax),kg-CO2e/tkm,0.00526,,UBA,2020,DE,2021/09/15,Emission intensity for overseas vessel energy demand and emissions data for crude oil tankers and bulk carriers (liquid and dry) of 80000 - 200000 dwt (Aframax - Suezmax) modeled after Lloyds (2009) data. Provided by the Öko-Institut.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={C04CE7CC-1470-4A37-BA94-E97669CC248A}
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1243]: @greenercode the details in the description here do't appear to match the ID and name (eg. the dwt provided in the ID is outside the range of dwt provided in the description) - could you please check this and the other UBA lines?

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 5 above)
Comment (greenercode) [reply OpenEmissionFactorsDB.csv]: Addressed, are we good to ship now? :)

**Review by AnastasiaLobanova — DISMISSED** [review]

**Review by patstirling — APPROVED** [review]

**Review by patstirling — DISMISSED** [review]

**Review by AnastasiaLobanova — APPROVED** [review]

**Pull request merged**